fw/access_log.c

```diff
@@ -413,21 +543,89 @@ do_access_log(TfwHttpResp *resp)
 			  TFW_HTTP_RESP_CUT_BODY_SZ(resp));
 }
 
+static int
+cfg_access_log_set(TfwCfgSpec *cs, TfwCfgEntry *ce)
+{
+	int i;
+	const char *val;
+
+	TFW_CFG_CHECK_VAL_N(>, 0, cs, ce);
+	TFW_CFG_CHECK_NO_ATTRS(cs, ce);
+
+	access_log_type = ACCESS_LOG_OFF;
+
+	TFW_CFG_ENTRY_FOR_EACH_VAL(ce, i, val) {
+		if (strcasecmp(val, "off") == 0)
```

Comment:
It seems it works incorrect. For example if we write `access log dmesg off`; we have dmesg logging. If we write
`acess_log off dmesg;` we have no logging. I think if off set with some other option we should return -EINVAL

Reply:
Fixed.